src/transaction/log_page_buffer.c

```diff
@@ -2115,6 +2115,10 @@ logpb_request_log_page_from_page_server (LOG_PAGEID log_pageid, LOG_PAGE * log_p
   // client side communication to page server error
   if (error_code != NO_ERROR)
     {
+      if (error_code == ER_NO_PAGE_SERVER_CONNECTION)
```

Comment:
isn't it better to move this after the `if (perform_logging)` scope?
same above

Reply:
Done